From 81b7b0eeee850d8adb301dbf23e6e523d20ce547 Mon Sep 17 00:00:00 2001
From: Anil Dsouza <anil.dsouza@einfochips.com>
Date: Mon, 10 Jan 2022 14:14:11 +0530
Subject: [PATCH 3/5] arch: arm: configs: Enable AP1302 by default for AV96

This patch enables AP1302 by default as loadable module for AV96.
This patch also make ST mipid02 as loadable module.

Change-Id: Ibf3b490e355deb7775c6a3ff464c40f5030d0fa9
Signed-off-by: Anil Dsouza <anil.dsouza@einfochips.com>
---
 arch/arm/configs/stm32mp1_dhsom_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/stm32mp1_dhsom_defconfig b/arch/arm/configs/stm32mp1_dhsom_defconfig
index f594e8bbc..5f80a9e83 100644
--- a/arch/arm/configs/stm32mp1_dhsom_defconfig
+++ b/arch/arm/configs/stm32mp1_dhsom_defconfig
@@ -262,7 +262,8 @@ CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_VIDEO_STM32_DCMI=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 CONFIG_VIDEO_OV5640=y
-CONFIG_VIDEO_ST_MIPID02=y
+CONFIG_VIDEO_ST_MIPID02=m
+CONFIG_VIDEO_AP1302=m
 CONFIG_DRM=y
 CONFIG_DRM_LVDS_CODEC=y
 CONFIG_DRM_STM=y
-- 
2.17.1

